accepted/2020/windows-specific-apis/windows-specific-apis.md

```diff
@@ -0,0 +1,378 @@
+* **CommunicationException**
+    - `.ctor(SerializationInfo, StreamingContext)`
+* **CommunicationObjectAbortedException**
+    - `.ctor(SerializationInfo, StreamingContext)`
+* **CommunicationObjectFaultedException**
+    - `.ctor(SerializationInfo, StreamingContext)`
+* **DuplexChannelFactory**
+    - `.ctor(InstanceContext, Binding, EndpointAddress)`
+    - `.ctor(InstanceContext, Binding, String)`
+    - `.ctor(InstanceContext, String)`
+    - `.ctor(InstanceContext, String, EndpointAddress)`
+* **DuplexClientBase**
+    - `.ctor(InstanceContext)`
+    - `.ctor(InstanceContext, String)`
+    - `.ctor(InstanceContext, String, EndpointAddress)`
+    - `.ctor(InstanceContext, String, String)`
+* **EndpointAddress**
+    - `ApplyTo(Message)`
+* **EndpointIdentity**
+    - `Equals(Object)`
+    - `GetHashCode()`
+* **EndpointNotFoundException**
+    - `.ctor(SerializationInfo, StreamingContext)`
+* **FaultException**
+    - `.ctor(SerializationInfo, StreamingContext)`
+* **InvalidMessageContractException**
+    - `.ctor(SerializationInfo, StreamingContext)`
+* **MessageHeaderException**
+    - `.ctor(SerializationInfo, StreamingContext)`
+* **MessageSecurityOverTcp**
+    - `get_ClientCredentialType()`
+    - `set_ClientCredentialType(MessageCredentialType)`
+* **NetHttpBinding**
+    - `CreateBindingElements()`
+* **NetHttpsBinding**
+    - `.ctor()`
+    - `.ctor(BasicHttpsSecurityMode)`
+    - `CreateBindingElements()`
+* **NetTcpBinding**
+    - `.ctor(String)`
+    - `CreateBindingElements()`
+* **ProtocolException**
+    - `.ctor(SerializationInfo, StreamingContext)`
+* **QuotaExceededException**
+    - `.ctor(SerializationInfo, StreamingContext)`
+* **ServerTooBusyException**
+    - `.ctor(SerializationInfo, StreamingContext)`
+* **ServiceActivationException**
+    - `.ctor(SerializationInfo, StreamingContext)`
+
+### System.ServiceModel.Channels
+
+* **AddressHeader**
+    - `Equals(Object)`
+    - `GetHashCode()`
+* **MessageHeaders**
+    - `set_To(Uri)`
+* **SecurityBindingElement**
+    - `CreateSecureConversationBindingElement(SecurityBindingElement)`
+* **TransportSecurityBindingElement**
+    - `GetProperty<T>(BindingContext)`
+* **WebSocketTransportSettings**
+    - `get_DisablePayloadMasking()`
+    - `set_DisablePayloadMasking(Boolean)`
+
+### System.ServiceModel.Security
+
+* **MessageSecurityException**
+    - `.ctor(SerializationInfo, StreamingContext)`
+* **SecurityAccessDeniedException**
+    - `.ctor(SerializationInfo, StreamingContext)`
+* **SecurityNegotiationException**
+    - `.ctor(SerializationInfo, StreamingContext)`
+* **X509ServiceCertificateAuthentication**
+    - `set_CertificateValidationMode(X509CertificateValidationMode)`
+
+### System.Threading
+
+* **EventWaitHandle**
+    - `OpenExisting(String)`
+    - `TryOpenExisting(String, EventWaitHandle)`
+* **Semaphore**
+    - `OpenExisting(String)`
+    - `TryOpenExisting(String, Semaphore)`
+* **Thread**
+    - `SetApartmentState(ApartmentState)`
```

Comment:
what about the `ApartmentState` (it's obsolete) and `TrySetApartmentState`? the seem to do the same thing

Reply:
@stephentoub looked it up. `TrySetApartmentState` always returns `false` on non-Windows and `SetAparmentState` throws if `TrySetApartmentState` returns `false`. Hence, `SetAparmentState` should be marked by `TrySetApartmentState` shouldn't.

Reply:
@terrajobst thanks!